### ISSUE TITLE
Feature Request: User with worker role is not able to select the checklist items

### ISSUE THREAD

**Issue by mweiss237** [wekan/wekan#3307]
Hi,
I have a use case where the task of single users is shown in the checklists. Now the users should have the possibility to complete thesetasks but not to delete them. First I thought about the role of the "worker" but as a worker you cannot check off any tasks. As a user with role "normal" you have the possibility to delete the checklist items, which they shouldn't be able to do...
Wouldn't it make sense to give a worker the permission to do this? 
Maybe this could be solved with another role like "contributor" or even better with the ability to create custom roles.

Thanks!

**Comment by xet7** [comment]
@nickbe 

What do you think? Can I make change to current worker role that it can check checklist items?

**Comment by gustavengstrom** [comment]
I agree! I also think it would be very useful with another role like "contributor" or preferably as @mweiss237 suggest, the ability to create custom roles...
